tlatools/org.lamport.tlatools/customBuild.xml

```diff
@@ -98,20 +98,11 @@
 
 	<!-- Runs JavaCC to generate the TLA+ parser files. -->
 	<target name="generate">
-		<tempfile property="javacc-tmpdir" destdir="${java.io.tmpdir}" prefix="javacc_" />
-		<mkdir dir="${javacc-tmpdir}" />
 		<java classpath="lib/javacc-4.0.jar" classname="javacc" fork="true" failonerror="true">
-			<arg value="-OUTPUT_DIRECTORY=${javacc-tmpdir}" />
+      <arg value="-OUTPUT_DIRECTORY=src/tla2sany/parser" />
```

Comment:
The surrounding lines use tabs, but this line is using spaces.

Reply:
Good catch!